### DISCUSSION TITLE
Using ReduceLROnPlateau

### DISCUSSION THREAD

**Discussion by DannyFGitHub** [ContinualAI/avalanche#854] [Q&A] [answered]
Is there a built in way of using pytorch's ReduceLROnPlateau on the LRSchedulerPlugin. Or does Avalanche have a built-in way of implementing ReduceLROnPlateau?
I've come accross a few issues such as : step() takes a metric where the plugin does not pass a metric.

**Comment by vlomonaco** [comment, marked as answer]
Hi @DannyFGitHub! We started working on this here: #858!

**Reply by DannyFGitHub** [reply]
Awesome! 🙏
